src/common/runtime-upgrades/remove-httpbin-application.ts

```diff
@@ -0,0 +1,24 @@
+import { terminal } from '../debug'
+import { k8s } from '../k8s'
+import { ARGOCD_APP_PARAMS } from '../constants'
+
+const httpbinAppName = 'httpbin-httpbin'
+
+export async function removeHttpBinApplication() {
+  const d = terminal('removeHttpBinApplication')
+  try {
+    // Delete httpbin ArgoCD application
+    const custom = k8s.custom()
+    await custom.deleteNamespacedCustomObject({
+      ...ARGOCD_APP_PARAMS,
+      name: httpbinAppName,
+    })
+    d.info(`Deleted ${httpbinAppName} ArgoCD application.`)
+  } catch (err: any) {
+    if (err?.statusCode === 404 || err?.response?.statusCode === 404) {
+      d.info(`${httpbinAppName} application not found in ${ARGOCD_APP_PARAMS.namespace} namespace, skipping deletion.`)
```

Comment:
Still would suggest this to be a debug message. Since it has been deprecated for a while, it should rather be the exception that it is installed.
```suggestion
      d.debug(`${httpbinAppName} application not found in ${ARGOCD_APP_PARAMS.namespace} namespace, skipping deletion.`)
```